kratos/utilities/dof_utilities/elimination_build_dof_array_utility.h

```diff
@@ -0,0 +1,227 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
```

Comment:
As you are taking code from old classes I would include the authors of the older files; AKA Riccardo and Pooyan